docs/docs/account/user-permissions.mdx

```diff
@@ -140,44 +140,44 @@ When creating and editing custom roles, organizations have the ability to select
-|                       | TeamManagementFullAccess  | Invite users, delete users, change user roles, add/remove users from teams.                                                  | readData, manageTeam                                                           |
-|                       | CustomRolesFullAccess     | Create, edit, and delete projects                                                                                            | readData, manageProjects                                                       |
-|                       | ProjectsFullAccess        | Create, edit, and delete tags                                                                                                | readData, manageTags                                                           |
-|                       | TagsFullAccess            | Create, edit, and delete API secret keys. Not required to create Personal Access Tokens.                                     | readData, manageApiKeys                                                        |
-|                       | APIKeysFullAccess         | Set up and configure integrations - GitHub, Vercel, etc.                                                                     | readData, manageIntegrations                                                   |
-|                       | IntegrationsFullAccess    | Create, edit, and delete event-based webhooks. Used for Slack/Discord notifications.                                         | readData, manageEventWebhooks, viewAuditLog                                    |
-|                       | EventWebhooksFullAccess   | View and edit license key. View invoices and update billing info.                                                            | readData, manageBilling                                                        |
-|                       | BillingFullAccess         | View and export audit logs                                                                                                   | readData, viewAuditLog                                                         |
-|                       | AuditLogsFullAccess       | Create, edit, and delete custom roles                                                                                        | readData, manageTeam, manageCustomRoles                                        |
-|                       | DecisionCriteriaFullAccess | Create, edit, and delete decision criteria                                                                                   | readData, manageDecisionCriteria                                                 |
+| Policy Group          | Policy                     | Description                                                                                                                  | Permissions                                                                    |
```

Comment:
Hmm, didn't mean that this file got changed, but it autoformats to this, so might as well leave it